### PR TITLE
Add human readable error for missing codes

### PR DESCRIPTION
**Pull request by thi4go**
These error codes were recently added to pi: 60, 61 and 62.

This PR updates our human readable errors to add them.

### CODE REVIEW

File: src/helpers.js [hunk 1]
```diff
@@ -111,14 +111,16 @@ export const getHumanReadableError = (errorCode, errorContext = []) => {
...
+    54: "Invalid proposal vote parameters.",
+    55: "Email address is not verified.",
+    56: "Invalid user ID.",
+    57: "Invalid like comment action.",
+    58: "Invalid proposal censorship token.",
+    59: "Email address is already verified.",
+    60: "No changes were found in the proposal.",
+    61: "Maximum proposal page size exceeded.",
+    62: "That is a duplicate comment.",
+    85: "The provided password was invalid.",
```
Comment (lukebp) [inline src/helpers.js]:
```suggestion
```

Error code `85` should be removed and error code `1` should be changed to:
`    1: "The provided password was invalid.",`

Comment (thi4go) [reply src/helpers.js]: Fixed

**Comment by fernandoabolafio** [comment]
@thi4go please fix the conflicts

**Comment by thi4go** [comment]
@fernandoabolafio fixed!

**Pull request merged**